seqr/views/utils/project_context_utils.py

```diff
@@ -141,6 +141,15 @@ def add_project_tag_types(projects_by_guid):
             project_guid = vtt.project.guid if vtt.project else None
             project_tag_types[project_guid].append(variant_tag_types_by_guid[vtt.guid])
 
+    project_tag_types[None].append({
```

Comment:
Why is the new tag added to the key `None` list?

Reply:
because this is a dict that groups tags by project, and uses None as the key for tags that go to every project, which is what we want in this case